src/ec_environment.F

```diff
@@ -59,18 +74,20 @@ MODULE ec_environment
 !> \param qs_env ...
 !> \param ec_env the object to create
 !> \param dft_section ...
+!> \param input ...
```

Comment:
Please use a more descriptive name than just `input`. This is a `force_eval` section, correct?